.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,61 @@
+#### What type of PR is this?
+
+<!--
```

Comment:
I think there are potentially other areas of value in this repo for PRs, like which component the PR applies to (e.g, `cluster-autoscaler` `vertical-pod-autoscaler`, `charts` etc., and potentially which cloud-provider it touches for CA contributions.